src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/TrainerAPI.cs

```diff
@@ -32,8 +32,6 @@ public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymou
             var created = await repository.CreateAsync(new Trainer { FirstName = trainer.FirstName, LasttName = trainer.LasttName, EmailAddress = trainer.EmailAddress });
```

Comment:
This line can be simply written as `var created = await repository.CreateAsync(trainer);`

Reply:
Done